eng/scripts/Language-Settings.ps1

```diff
@@ -348,6 +369,43 @@ function Get-DocsCiLine ($item) {
   return $line
 }
 
+function EnsureCustomSource($package) {
+  # $PackageSourceOverride is a global variable provided in
+  # Update-DocsMsPackages.ps1. Its value can set a "customSource" property.
+  # If it is empty then the property is not overridden.
+  $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+  if (!$customPackageSource) {
+    return $package
+  }
+
+  if (!(Get-PackageSource -Name CustomPackageSource -ErrorAction SilentlyContinue)) {
```

Comment:
```suggestion
  if (!(Get-PackageSource -Name CustomPackageSource -ErrorAction Ignore)) {
```
I think you want Ignore here to avoid getting error messages still.

Reply:
`SilentlyContinue` did work but `Ignore` is a better option.